### PR TITLE
Added support to tail logs for aio app run command

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Get action logs for aio app run command. Both local and remote actions
## How Has This Been Tested?
execute
aio app run  and aio app run --local commands

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: scripts/dev.js [hunk 1]
```diff
@@ -184,13 +185,30 @@ class ActionServer extends BaseScript {
         resources.dummyProc = execa('node')
       }
       this.emit('progress', 'press CTRL+C to terminate dev environment')
+
+      if (hasBackend) {
+        // fetch action logs
+        const AppScripts = require('../index.js')
+        const scripts = AppScripts({ listeners: {} })
+        this.getLogs(scripts.logs, this.getLogs)
```
Comment (moritzraho) [inline scripts/dev.js]: is this polling for logs?

File: scripts/dev.js [hunk 2]
```diff
@@ -184,13 +185,30 @@ class ActionServer extends BaseScript {
...
     return frontEndUrl
   }
 
+  async getLogs (logScript, cb) {
+    try {
+      await logScript([], {})
+    } catch (e) {
+      aioLogger.error('Error while fetching action logs ' + e)
+    } finally {
+      setTimeout(function () { cb(logScript, cb) }, fetchLogInterval)
```
Comment (moritzraho) [inline scripts/dev.js]: what if there were 5 invocations in between two fetch log interval?

Comment (sandeep-paliwal) [reply scripts/dev.js]: I have made a few changes to log function. Now we fetch default 30 activations but keep track of last activation which was logged. This way we should not miss any activations, plus we don't end up repeat logging them.

**Comment by shazron** [comment]
According to the logs it is suggesting you need `--detectOpenHandles` flag for Jest to detect the errors, I see https://github.com/adobe/aio-app-scripts/pull/99/commits/11fe3bed4e108a3437b41118c741a3c6d3213a4c but it's not reflected in the test run. Perhaps some merge weirdness.

**Comment by sandeep-paliwal** [comment]
The `--detectOpenHandles` didn't help much. I see the basic issue on windows is coverage not being 100. I have changed the coverage threshold so that the build doesn't fail. Looks like this issue with difference in coverage on Linux/windows will need more investigation.

**Comment by shazron** [comment]
I would leave it as failing and not change the threshold, it will just mask the issue. Can't make heads or tails of it when I looked. Someone needs to take a look with fresh eyes @purplecabbage ?

**Comment by moritzraho** [comment]
@sandeep-paliwal there are some conflicts now and the test fail because they don't finish, there is probably a hanging async test somewhere..

**Comment by sandeep-paliwal** [comment]
@moritzraho The conflict is resolved. The test pass now with complete coverage. But looks like jest exit has some issue.

**Comment by moritzraho** [comment]
lgtm!

**Pull request merged**